ios/RN/RNCamera.m

```diff
@@ -6,6 +6,7 @@
 #import <React/RCTLog.h>
 #import <React/RCTUtils.h>
 #import <React/UIView+React.h>
+#import <MobileCoreServices/MobileCoreServices.h>
```

Comment:
@sibelius  do you know if this import might add an extra framework requirement? Looks like the MobileCoreServices framework is always available for iOS applications (and it doesn't need to be added manually or anything), but just to be sure this won't cause any conflicts.

Reply:
this is available to everyone I guess